playground/optimize-deps/vite.config.js

```diff
@@ -19,6 +19,7 @@ export default defineConfig({
     include: [
       '@vitejs/test-dep-linked-include',
       '@vitejs/test-nested-exclude > @vitejs/test-nested-include',
+      '@vitejs/test-dep-css-require',
```

Comment:
Do we need to specify it explicitly here? I think the scanner should pick it up automatically in the `index.html`.

Reply:
Thanks for pointing that out. I didn't notice that.